plugins/Jobs.cpp

```diff
@@ -442,7 +465,7 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
             int64_t parentJobID = SContains(job, "parentJobID") ? SToInt(job["parentJobID"]) : 0;
             if (parentJobID) {
                 SQResult result;
-                if (!db.read("SELECT state, parentJobID FROM jobs WHERE jobID=" + SQ(parentJobID) + ";", result)) {
+                if (!db.read("SELECT state, parentJobID, retryAfter FROM jobs WHERE jobID=" + SQ(parentJobID) + ";", result)) {
```

Comment:
I'm not sure why this is here, removing it.

Reply:
This is here because auto-retrying parents shouldn't have children - https://github.com/Expensify/Bedrock/pull/169/files#diff-d116ebb4bb9e2a3fd799045c7b182707R372

Adding this back.  I think it disappeared in a bad merge with master when we added `parentJobID` to this query

Reply:
Never mind, I'm confusing myself.  We check this in `peek` and the code I linked to is alerting, not throwing.  Removing this